src/Controller/Admin/CategoryCrudController.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace App\Controller\Admin;
+
+use App\Entity\Category;
+use EasyCorp\Bundle\EasyAdminBundle\Config\Action;
+use EasyCorp\Bundle\EasyAdminBundle\Config\Actions;
+use EasyCorp\Bundle\EasyAdminBundle\Config\Crud;
+use EasyCorp\Bundle\EasyAdminBundle\Controller\AbstractCrudController;
+use EasyCorp\Bundle\EasyAdminBundle\Field\CollectionField;
+use EasyCorp\Bundle\EasyAdminBundle\Field\IdField;
+use EasyCorp\Bundle\EasyAdminBundle\Field\TextField;
+use EasyCorp\Bundle\EasyAdminBundle\Field\TimeField;
+
+class CategoryCrudController extends AbstractCrudController
+{
+    public static function getEntityFqcn(): string
+    {
+        return Category::class;
+    }
+
+    public function configureActions(Actions $actions): Actions
+    {
+        $actions
+            ->add(Crud::PAGE_INDEX, Action::DETAIL);
+        return parent::configureActions($actions); // TODO: Change the autogenerated stub
```

Comment:
Do this like shown on https://symfony.com/bundles/EasyAdminBundle/current/actions.html, i.e.

```php
        return $actions
            ->add(Crud::PAGE_INDEX, Action::DETAIL);
```